ansible/roles/operator-pipeline/templates/openshift/tasks/acquire-lock.yaml

```diff
@@ -68,4 +84,4 @@ spec:
         create_lease_yaml
         # Acquire Lease
         export -f acquire_lease
-        timeout $(params.timeout) bash -c "acquire_lease"
+        bash -c "acquire_lease"
```

Comment:
Having removed the timeout wrapper, I don't think you need to run `acquire_lease()` in a subshell anymore and as a result, the `export -f acquire_lease` and `export TOTAL_DURATION_IN_SECONDS` can also be dropped.